events/templates/events/events.html

```diff
@@ -185,6 +184,43 @@ <h1>Events</h1>
     <script src="{% static 'events/js/bootstrap.bundle.min.js' %}"></script>
     {% block extra_scripts %}
     <script>
+        function getQueryParameters() {
+    var queryString = window.location.search.slice(1);
+    var params = {};
+
+    if (queryString) {
+        var keyValuePairs = queryString.split('&');
+        keyValuePairs.forEach(function(keyValuePair) {
+        var pair = keyValuePair.split('=');
+        var key = decodeURIComponent(pair[0]);
+        var value = decodeURIComponent(pair[1]);
+        params[key] = value;
+        });
+    }
+
+    return params;
+    }
+        window.addEventListener('load', function() {
+        var queryParameters = getQueryParameters();
+        
+        // Populate the form fields if the query parameters exist
+        if (queryParameters.start_time) {
+            document.getElementById('start_time').value = queryParameters.start_time;
+        }
+        if (queryParameters.end_time) {
+            document.getElementById('end_time').value = queryParameters.end_time;
+        }
+        if (queryParameters.min_capacity) {
+            document.getElementById('min_capacity').value = queryParameters.min_capacity;
+        }
+        if (queryParameters.max_capacity) {
+            document.getElementById('max_capacity').value = queryParameters.max_capacity;
+        }
+        if (queryParameters.max_capacity) {
+            document.getElementById('search_name').value = queryParameters.search;
+        }
```

Comment:
is this intentional here to check for max_capacity when it seems to check for search_name?